lib/exchange_clients/bitfinex/util/filter_pairs.js

```diff
@@ -1,13 +1,13 @@
 'use strict'
 
-module.exports = function filterPairs (symbols, exclude) {
+module.exports = function filterPairs (symbols, exclude, reverse = false) {
   const sf = symbols.filter((el) => {
-    let include = true
+    let include = reverse === false
```

Comment:
Do we send the reverse value in any other format except boolean? If not, we could may be do `let include = !reverse` ?